experimental/segmenter/src/line.rs

```diff
@@ -159,6 +159,40 @@ pub type LineBreakIteratorUtf16<'l, 's> = LineBreakIterator<'l, 's, LineBreakTyp
 /// assert_eq!(&breakpoints, &[6, 11]);
 /// ```
 ///
+/// The segmenter returns mandatory breaks (as defined by [definition LD7][LD7] of
+/// Unicode Standard Annex #14, _Unicode Line Breaking Algorithm_) as well as
+/// line break opportunities ([definition LD3][LD3]).
+/// It does not distinguish them.  Callers requiring that distinction can check
+/// the Line_Break property of the code point preceding the break against those
+/// listed in rules [LB4][LB4] and [LB5][LB5], special-casing the end of text
+/// according to [LB3][LB3].
+///
+/// Note that contrary to the grapheme, word, and sentence segmenters, the
+/// breaks returned by this segmenter do not determine a partition of the text
+/// into meaningful segments.  In particular, there is no break opportunity at
+/// the start of text.
+///
+/// [LD3]: https://www.unicode.org/reports/tr14/#LD3
+/// [LD7]: https://www.unicode.org/reports/tr14/#LD7
+/// [LB3]: https://www.unicode.org/reports/tr14/#LB3
+/// [LB4]: https://www.unicode.org/reports/tr14/#LB4
+/// [LB5]: https://www.unicode.org/reports/tr14/#LB5
```

Comment:
thought (non-blocking): since the example has been moved to the bottom of the example section, it's probably worth moving to line 140, after the paragraph "Supports loading line break data....", if we feel this is an useful explanation that every line segmenter user should be aware of.